src/index.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ** This file is automatically generated by gapic-generator-typescript. **
+// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** All changes to this file may be overwritten. **
+
+import * as v1beta1 from './v1beta1';
+import * as v1 from './v1';
+
+module.exports = v1;
```

Comment:
I could be mistaken.  I don't think TypeScript will work correctly unless we use the syntax like:

```js
export {v1, v1beta1};
```

The other gymnastics we're doing with default exports hurt my brain.   @bcoe help?

Reply:
Agreed - we should not use `module.exports` in TypeScript code. @xiaozhenliu-gg5 those are manually written files, not generated, can you please change it to what Justin suggests?

Reply:
@JustinBeckwith I believe it will still work correctly :)  (TypeScript just copies those lines into the resulting `build/src/index.js`), but it's obviously not how it should work.

Reply:
Oh sorry I mistook the idea, sure, I will make the changes after the system-test commit is pushed, thanks